js/_website/src/lib/components/EventListeners.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts">
+	export let fns: any[];
+</script>
+
+
+<div id="event-listeners-description">
+    <h4 class="mt-8 text-xl text-orange-500 font-light group">
+        Description
+    </h4>
+    <p class="mb-2 text-lg text-gray-600">Event listeners are functions that are called when an event is triggered. Every event listener takes the same arguments, shown in the arguments table below.</p>
+    <p class="mb-2 text-lg text-gray-600"> The table below shows the event listeners supported by {fns[0].parent.replace("gradio.","")}.</p>
```

Comment:
Nit: could we make the component name monospace. Something like this might work:
```suggestion
    <p class="mb-2 text-lg text-gray-600"> The table below shows the event listeners supported by the `{fns[0].parent.replace("gradio.","")}` component.</p>
```